kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-planning/src/main/java/org/kie/server/services/taskassigning/planning/TaskAssigningConstants.java

```diff
@@ -36,6 +36,18 @@ private TaskAssigningConstants() {
      */
     public static final String JBPM_TASK_ASSIGNING_PROCESS_RUNTIME_PWD = "org.kie.server.taskAssigning.processRuntime.pwd";
 
+    /**
+     * Property for configuring the alias of the user password for connecting with the jBPM runtime in cases where the
+     * kie-server key store is used.
+     */
+    public static final String JBPM_TASK_ASSIGNING_KEY_STORE_PROCESS_RUNTIME_ALIAS = "kie.keystore.key.taskAssigning.processRuntime.alias";
+
+    /**
+     * Property for configuring the password for getting the key stored under the alias name
+     * JBPM_TASK_ASSIGNING_KEY_STORE_PROCESS_RUNTIME_PWD_ALIAS in cases where the kie-server key store is used.
```

Comment:
A typo? If this should refer to the constant defined above, there is extra 'PWD' in the name of the constant.